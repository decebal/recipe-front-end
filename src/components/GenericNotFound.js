import React from 'react';
import { Mug } from 'react-kawaii';
import { Headline, Paragraph } from './layout';

const GenericNotFound = () => (
  <div>
    <Headline>Not Found</Headline>
    <Paragraph>
      Apologise for the inconvenience, the current page does not exist yet.
    </Paragraph>
    <Mug size={170} mood="shocked" color="#A6E191" />
  </div>
);

export default GenericNotFound;
