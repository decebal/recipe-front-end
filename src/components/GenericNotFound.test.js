import React from 'react';
import renderer from 'react-test-renderer';
import GenericNotFound from './GenericNotFound';


test('GenericNotFound renders correctly', () => {
  const tree = renderer
    .create(<GenericNotFound />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
