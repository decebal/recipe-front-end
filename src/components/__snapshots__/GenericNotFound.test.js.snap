// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GenericNotFound renders correctly 1`] = `
<div>
  <h1
    className="sc-bxivhb hPRIrZ"
  >
    Not Found
  </h1>
  <p
    className="sc-ifAKCX mgcNx"
  >
    Apologise for the inconvenience, the current page does not exist yet.
  </p>
  <div
    style={
      Object {
        "position": "relative",
      }
    }
  >
    <svg
      height={170}
      viewBox="0 0 172 115"
      width={255}
      xmlns="http://www.w3.org/2000/svg"
      xmlnsXlink="http://www.w3.org/1999/xlink"
    >
      <g
        id="kawaii-mug"
      >
        <g
          fillRule="nonzero"
          id="kawaii-mug__body"
        >
          <path
            d="M17.591 22.258h21.5V5.565C39.091 2.49 41.718 0 44.955 0h121.181C169.374 0 172 2.49 172 5.565v70.483C172 97.526 153.588 115 130.955 115H80.136c-20.764 0-37.93-14.72-40.633-33.733C17.283 78.701 0 60.722 0 38.952c0-9.205 7.891-16.694 17.591-16.694zm20.627 47.704V33.387h-21.5c-3.233 0-5.863 2.496-5.863 5.565 0 15.485 11.835 28.37 27.363 31.01z"
            fill="#A6E191"
            id="kawaii-mug__shape"
          />
          <path
            d="M156.99 0h9.106C169.356 0 172 2.49 172 5.565v70.483C172 97.526 153.462 115 130.673 115h-16.89 7.784c22.79 0 41.327-17.474 41.327-38.952V5.565c0-3.074-2.644-5.565-5.903-5.565H106z"
            fill="#000"
            id="kawaii-mug__shadow"
            opacity=".1"
          />
        </g>
        <g
          id="kawaii-face"
          transform="translate(71 42)"
        >
          <defs>
            <path
              d="M1.45656876,3.14684877 C1.45656876,3.14684877 1.45656876,3.14684877 1.45656876,3.14684877 L0,3.14685315 C0,2.31818182 0.346033696,1.50734266 0.949429952,0.922027972 C1.55390756,0.335664336 2.38979521,0 3.2440659,0 L25.9525272,0 C26.8067979,0 27.6416041,0.335664336 28.2460818,0.922027972 C28.8505594,1.50734266 29.1965931,2.31818182 29.1965931,3.14685315 C29.1890236,5.85734266 28.240675,8.44825175 26.7127199,10.6814685 C25.1771954,12.9104895 23.0317865,14.8122378 20.4040931,16.0227273 C18.6544603,16.8251748 16.6809868,17.3087413 14.5982965,17.3076923 C11.4666916,17.3076923 8.61299495,16.2241259 6.33025392,14.5951049 C4.0399434,12.9587413 2.264358,10.779021 1.16245695,8.33811189 C0.431460764,6.70909091 0.0010813553,4.95314685 0,3.14685315 L1.45656896,3.14685315 Z"
              id="kawaii-face__path-1"
            />
          </defs>
          <g
            id="kawaii-face__mouth"
            transform="translate(18.000000, 16.000000)"
          >
            <ellipse
              cx="15"
              cy="14"
              fill="#000000"
              id="kawaii-face__mouth__shocked"
              rx="9"
              ry="10"
            />
          </g>
          <g
            fill="#000000"
            id="kawaii-face__blush"
            opacity="0.2"
            transform="translate(0.000000, 15.000000)"
          >
            <circle
              cx="3"
              cy="3"
              id="Oval"
              r="3"
            />
            <circle
              cx="63"
              cy="3"
              id="Oval"
              r="3"
            />
          </g>
          <g
            fill="#000000"
            id="kawaii-face__eyes"
            transform="translate(2.000000, 0.000000)"
          >
            <g
              id="kawaii-face__eyes__circle"
              transform="translate(1.000000, 2.000000)"
            >
              <circle
                cx="4.5"
                cy="4.5"
                id="Oval-3"
                r="4.5"
              />
              <circle
                cx="56.5"
                cy="4.5"
                id="Oval-3"
                r="4.5"
              />
            </g>
          </g>
        </g>
      </g>
    </svg>
  </div>
</div>
`;
